src/GraphQL.Tests/Types/AutoRegisteringInputObjectGraphTypeTests.cs

```diff
@@ -288,7 +288,7 @@ private class FieldTests
             public ICollection CollectionField { get; set; } = null!;
             public IEnumerable? NullableEnumerableField { get; set; }
             public ICollection? NullableCollectionField { get; set; }
-            public int[]?[]? ListOfListOfIntsField { get; set; }
+            public int?[]?[]? ListOfListOfIntsField { get; set; }
```

Comment:
This is an example of the buggy NRT code by MS.  `int[]?[]?` does not parse correctly on .NET 6, but with the patched code present in the source-only repo, it works correctly on .NET 5 and will also on .NET 7.

However, `int?[]?[]?` parses the same on both .NET 5 and .NET 6, so I just changed the test to that.